apps/server-asset-sg/src/app/jwt/jwt-middleware.ts

```diff
@@ -81,6 +82,16 @@ export class JwtMiddleware implements NestMiddleware {
         );
     }
 
+  private isAuthorizedByGroupE(jwt: Jwt): E.Either<Error, Jwt> {
+      const authorizedGroups = process.env.OAUTH_AUTHORIZED_GROUPS?.split(',');
```

Comment:
- Trim each group so that checks don't fail when we have spaces between groups in our config.
- Lowercase each group so that checks don't fail when we use different casing in our config.